### PR TITLE
sl-185: Fix pop up in account settings - login

### PR DESCRIPTION
**Pull request by Izdormu**
fixed modal move when click esc

### CODE REVIEW

File: frontend/src/libs/components/modal/styles.module.scss [hunk 1]
```diff
@@ -4,6 +4,7 @@
   align-items: center;
   background: none;
   border: none;
+  outline: none;
```
Comment (what1s1ove) [inline frontend/src/libs/components/modal/styles.module.scss:7]:
```suggestion
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]
merge development into your branch

File: frontend/src/libs/components/modal/styles.module.scss (hunk 1 above)
Comment (Izdormu) [reply frontend/src/libs/components/modal/styles.module.scss:7]: if i not add this,when you click esc you will be have border of you browser
outline

Comment (Izdormu) [reply frontend/src/libs/components/modal/styles.module.scss:7]: and do i need fix problem that write me linter and add reference.current  inside useEffect like const (const modalElement = reference.current) and after in return will remove  event of him

**Review by andanf-e — COMMENTED** [review]
Pay attention on CI errors and warns

File: frontend/src/libs/components/modal/styles.module.scss (hunk 1 above)
Comment (andanf-e) [reply frontend/src/libs/components/modal/styles.module.scss:7]: https://www.ibm.com/docs/en/raa/6.1?topic=metrics-cyclomatic-complexity

Comment (what1s1ove) [reply frontend/src/libs/components/modal/styles.module.scss:7]: you should not delete this

**Review by what1s1ove — COMMENTED** [review]
As I can see, this ticket was closed - https://github.com/BinaryStudioAcademy/bsa-winter-2022-2023-simpleland/issues/185

**Pull request closed without merging**